src/js/module.js

```diff
@@ -189,7 +189,15 @@ iotjs_module_t.load = function(id, parent) {
 
   module.filename = modPath;
   module.dirs = [modPath.substring(0, modPath.lastIndexOf('/') + 1)];
-  module.compile();
+
+  var ext = modPath.substr(modPath.lastIndexOf('.') + 1);
+
+  if(ext === 'js') {
+    module.compile();
+  } else if(ext === 'json') {
+    var source = process.readSource(modPath);
+    module.exports = JSON.parse(source);
```

Comment:
Check if it is better to add error/exception handling on JSON parse.
If needed, prepare negative test json entry to test.

Reply:
A few people are working on it about what you have pointed out.